### DISCUSSION TITLE
Example for Salvo

### DISCUSSION THREAD

**Discussion by josejachuf** [juhaku/utoipa#399] [Q&A] [answered]
You could please provide an example for salvo web framework?

https://salvo.rs/

**Comment by juhaku** [comment, marked as answer]
Probably eventually yes, but for now I have been focusing on more popular libraries. And currently the main focus is in building the acutal library itself. 

To make an example for salvo would take some time from me to get it done since I am not familiar with salvo itself. It eventually would look quite close to warp or tide example. Since they are also frameworks that does not have boilerplate free integration with Swagger UI. If request towards salvo example would increase maybe it could be hurried a bit.

**Reply by josejachuf** [reply]
Thanks!

**Comment by josejachuf** [comment]
Hi @juhaku,
 
This [1] is an (draft) example of how to use it with Salvo.

Best Regards
Jose

[1] https://github.com/josejachuf/salvo-todos-api

**Reply by juhaku** [reply]
Very  nice :+1:
